ovara-backend/src/main/scala/fi/oph/ovara/backend/utils/GenericOvaraFormats.scala

```diff
@@ -0,0 +1,47 @@
+package fi.oph.ovara.backend.utils
+
+import fi.oph.ovara.backend.domain.Kieli
+import org.json4s.JsonAST.JString
+import org.json4s.jackson.Serialization.write
+import org.json4s.{CustomKeySerializer, CustomSerializer, DefaultFormats, Formats, JNull, MappingException}
+
+import scala.collection.Seq
+import scala.util.control.NonFatal
+
+trait GenericOvaraJsonFormats extends GenericOvaraFormats {
```

Comment:
En tiedä onko tuo json4s tarpeen jos on monimutkaisempaa rakennetta, mutta sen kanssa on ollut vähän ähräämistä. Käytin viestinvälityspalvelussa scala-toolkitin uPickle-kirjastoa ja json-olio-serialisoinnit (ja vastaavasti toisinpäin parsimiset) meni mun mielestä tosi näppärästi. 
Ks. https://docs.scala-lang.org/toolkit/json-serialize.html